ts/screens/wallet/AddPaymentMethodScreen.tsx

```diff
@@ -79,6 +79,12 @@ const getpaymentMethods = (props: Props): ReadonlyArray<IPaymentMethod> => [
       : "notImplemented",
     section: "digital_payments"
   },
+  {
+    name: I18n.t("wallet.methods.loyalty.name"),
+    description: I18n.t("wallet.methods.loyalty.description"),
+    status: "incoming",
+    section: "cashback",
```

Comment:
This is not correct. 
The **section** field enable the `BadgeStatus` that allow us to inform the users about problems: https://github.com/pagopa/io-app/blob/4e4e1b6640f0d8f996debf3411b86f86456f1030/ts/components/wallet/PaymentMethodsList.tsx#L93
You should add the `loyalty` section in `ts/types/backendStatus.ts`